twitterscraper/query.py

```diff
@@ -31,11 +30,12 @@ def query_single_page(url, html_response=True, retry=10):
     headers = {'User-Agent': random.choice(HEADERS_LIST)}
 
     try:
-        response = requests.get(url, headers=headers)
+        sess = requests.Session()
+        response = sess.get(url, headers=headers)
         if html_response:
             html = response.text
         else:
-            json_resp = response.json()
+            json_resp = ujson.loads(response.text)
```

Comment:
What is the difference between json.loads() and ujson.loads() ? If there is no clear reason for using ujson instead of json, I prefer the usage of json.